content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md

```diff
@@ -7,10 +7,10 @@ popular: false
 
 # How to manage suppliers proposals?
 ## Products
-When a product is completed on the Supplier Onboarder (ie [all attributes defined as required for the channel Onboarder](/onboarder/articles/define-product-requirements.html) are filled in by the supplier), you receive a proposal in your PIM.
+When a product is completed for a locale on the Supplier Onboarder, you receive a proposal in your PIM. For example, when [all attributes defined as required for the channel Onboarder on a specific locale](/onboarder/articles/define-product-requirements.html) are filled in by the supplier. 
 
 :::info
-A product updated by one of your supplier will never impact directly your PIM. A review step is mandatory so that you can evaluate the proposition.
+A product updated by one of your supplier will never impact directly your PIM. A review step is mandatory so that you can evaluate the proposal.
```

Comment:
```suggestion
A product updated by one of your suppliers will never impact directly your PIM. A review step is mandatory so that you can evaluate the proposal.
```